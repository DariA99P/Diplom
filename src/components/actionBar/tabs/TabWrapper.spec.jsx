import React from 'react';
import { shallow } from 'enzyme';

import TabWrapper from '.';

describe('<TabWrapper />', () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<TabWrapper />);
  });

  describe('On rendering', () => {
    it('should render properly', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
