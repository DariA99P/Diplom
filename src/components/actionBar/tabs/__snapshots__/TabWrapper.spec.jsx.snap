// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabWrapper /> On rendering should render properly 1`] = `
<div
  className="tabWrapper"
>
  <Menu
    className="tabWrapperMenu"
    focusable={false}
    mode="horizontal"
    onClick={[Function]}
    theme="light"
  >
    <MenuItem>
      Index
    </MenuItem>
    <MenuItem>
      Tab 1
    </MenuItem>
    <MenuItem>
      Tab 2
    </MenuItem>
    <MenuItem>
      Tab 3
    </MenuItem>
  </Menu>
</div>
`;
